File: untitled1/src/main/kotlin/makePrimeNumber.cpp
#include <vector>
#include <iostream>
using namespace std;
int isPrime(int sum)
{
    for(int i = 2 ; i < sum; i++)
        if(sum%i == 0) return false;
    return true;
}

int solution(vector<int> nums) {
    int answer = -1;

    // [실행] 버튼을 누르면 출력 값을 볼 수 있습니다.
    // cout << "Hello Cpp" << endl;

    int sum = 0;
    int count = 0;
    for(int i = 0 ; i < nums.size(); i++)
    {
        for(int j = i+1 ; j < nums.size(); j++ )
        {
            for(int k = j+1 ; k < nums.size(); k++)
            {
                sum = nums.at(i)+nums.at(j)+nums.at(k);

                //for(int x = 2 ; x < sum ; x++)
                //    if(sum % x == 0)
                //        break;
                // count++;
                if(isPrime(sum)) count++;
            }
        }
    }
    answer = count;

    return answer;
}

File: untitled1/src/main/kotlin/makePrimeNumber.kt
import kotlin.math.sqrt

class Solution {
    fun solution(nums: IntArray): Int {
        var count = 0

        for(i in 0 until nums.size-2){
            for(j in i+1 until nums.size-1){
                for(k in j+1 until nums.size){
                    val sum = nums[i]+nums[j]+nums[k]
                    if(isPrime(sum)) count++
                }
            }
        }
        return count
    }
    fun isPrime(num: Int): Boolean{
        if(num < 2) return false
        for(i in 2..sqrt(num.toDouble()).toInt()){
            if(num%i == 0) return false
        }
        return true
    }
}